chapters/connectors.tex

```diff
@@ -917,10 +943,11 @@ \subsection{Generation of Connection Graph Equations}\label{generation-of-connec
   If there are n selected roots in a subgraph, then optional spanning-tree edges have to be removed such that the result shall be a set of n spanning trees with the selected root nodes as roots.
```

Comment:
Missing math style, and avoid saying _if_ for something which isn't a condition:
```suggestion
  For a subgraph with $n$ selected roots, optional spanning-tree edges are removed such that the result is a set of $n$ spanning trees with the selected root nodes as roots.
```